frontend/src/container/NewDashboard/DashboardVariablesSelection/VariableItem.tsx

```diff
@@ -141,9 +141,26 @@ function VariableItem({
 			}
 	};
 
-	const selectValue = variableData.allSelected
-		? 'ALL'
-		: variableData.selectedValue?.toString() || '';
+	const getSelectValue = (): string | number | (number | string)[] => {
```

Comment:
```suggestion
	const getSelectValue = (): IDashboardVariable['selectedValue'] => {
```

Reply:
`IDashboardVariable['selectedValue']` contains `boolean` (and `boolean[]`), which is not allowed for `<Select value=`, as you can see here: https://ant.design/components/select#api.

That's the reason why the method converts booleans to strings.

This would result in a type mismatch.

Reply:
This would result in a type mismatch, because `IDashboardVariable['selectedValue']` contains `boolean` (and depending on which package is imported also `number`) which is not allowed in the select component for the value attribute.